### DISCUSSION TITLE
Can just use to compile&run text-based stuff, or can do Vulkan/glfw?

### DISCUSSION THREAD

**Discussion by ETERNALBLUEbullrun** [ish-app/ish#2391] [Ideas]
From https://github.com/ish-app/ish/discussions/2388 : can not produce either for iOS App Store (can just run stuff through Alpine Linux sandbox): question is, must use kvm/Qemu to do more than text, or, sandbox runs visual programs (such as Vulkan programs)?
https://github.com/glfw/glfw <- can do this?
https://github.com/KhronosGroup/Vulkan-Hpp <- can do this?
https://github.com/nigels-com/glew <- can do this?

**Comment by cOborski** [comment]
Vulkan doesn’t support ARM or Apple Silicon
